wandb/agents/agent.py

```diff
@@ -249,7 +249,8 @@ def _run_job(self, job):
             )
             config_util.save_config_file_from_dict(config_file, job.config)
             os.environ[wandb.env.RUN_ID] = run_id
-            os.environ[wandb.env.CONFIG_PATHS] = config_file
+            os.environ[wandb.env.DIR] = os.path.abspath(os.getcwd())
```

Comment:
Shouldn't we respect wandb.env.DIR if the user has set it in their environment before running an agent?